cmd/snap/wait.go

```diff
@@ -38,7 +38,8 @@ var (
 
 type mustWaitMixin struct {
 	clientMixin
-	skipAbort bool
+	skipAbort  bool
```

Comment:
Could you add comments as to what those fields control please.

Reply:
added